libbeat/_meta/fields.ecs.yml

```diff
@@ -613,6 +613,8 @@
       type: long
       format: duration
       input_format: nanoseconds
+      output_format: asMilliseconds
+      output_precision: 1
```

Comment:
Just as a side note, we previously had some confusion around showing durations that are stored and declared as nanoseconds in a different format (as this also changes format in discover), see https://github.com/elastic/apm-server/pull/1717